src/Endpoints/Users/WebhookSubscriptions/SubscribedActivity.php

```diff
@@ -0,0 +1,26 @@
+<?php
+declare(strict_types=1);
+
+namespace EoneoPay\PhpSdk\Endpoints\Users\WebhookSubscriptions;
+
+use EoneoPay\PhpSdk\Endpoints\Webhook;
+use EoneoPay\PhpSdk\Traits\Users\Webhook\SubscribedActivityTrait;
+use LoyaltyCorp\SdkBlueprint\Sdk\Entity;
+
+/**
+ * @method string|null getActivity()
+ * @method Webhook|null getUserWebhook()
+ * @method $this setActivity(string $activity)
+ */
+class SubscribedActivity extends Entity
+{
+    use SubscribedActivityTrait;
+
+    /**
+     * {@inheritdoc}
+     */
+    public function uris(): array
+    {
+        // There is no action directly on Subscribed Activity.
```

Comment:
This method should return an (empty) array because the return-type is hinted, otherwise it will result in a fatal error/exception when called

Reply:
True.. 👍